Bottom: c4ccca2e508fe43ce412d46aca94a2f5d46e20e5
Top:    613bff9fa23976d612dac1e6b72b3c97b285d2d0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-24 23:56:30 +0100

Refresh of temp

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 78cf91c..17b0934 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1850,11 +1850,7 @@ ScrollAction (int argc, char **argv, Coord x, Coord y)
   else
     AFAIL (scroll);
 
-//  notify_crosshair_change (false);
   ghid_port_ranges_pan (dx, dy, TRUE);
-//  MoveCrosshairRelative (dx, dy);
-//  AdjustAttachedObjects ();
-//  notify_crosshair_change (true);
 
   return 0;
 }
